app/locales/en/translations.js

```diff
@@ -362,6 +362,7 @@ export default {
     }
   },
   labels: {
+    currency_symbol: '$',
```

Comment:
this label is unused and also doesn't following naming standards.  Should be currencySymbol (if it was used).